test/integration/suites/batchx509SVID/01-setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo "RUNDIR: ${RUNDIR}"
+
+"${ROOTDIR}/setup/x509pop/setup.sh" conf/server conf/agent
+
+"${ROOTDIR}/setup/debugserver/build.sh" "${RUNDIR}/conf/server/debugclient"
+"${ROOTDIR}/setup/debugagent/build.sh" "${RUNDIR}/conf/agent/debugclient"
+
+
+echo "Starting SPIRE server..."
+spire-server run -config conf/server/server.conf > ${RUNDIR}/spire-server.log 2>&1 &
```

Comment:
We should follow the containerized model used throughout the integration framework. Presently, `make images` is broken in rootless container environments. See https://github.com/spiffe/spire/issues/5140.

Reply:
Per discussion in Contributor sync, maintainers said that it's OK to use this non-containerized style for direct checking of APIs. Ignore my previous comment here.